### DISCUSSION TITLE
Card Popup Archetype Stats

### DISCUSSION THREAD

**Discussion by FiYir** [unrealities/MTGA_Draft_17Lands#32] [Ideas]
Inspired by the LimitedGrades card popup, we could add archetype stats to our card's popup window. It could look like the image below.

The app would show all non-zero GIHWR win rate values for that card. If you pick `Ratings` or `Grades` win rate formats, it could also display those values in parentheses.

![image](https://github.com/unrealities/MTGA_Draft_17Lands/assets/151777715/0b4fa050-2659-46e0-8929-10b72c3d23ec)

**Comment by FiYir** [comment]
Here are some images of my progress. Depending on the `Win Rate Format,` the new table in the popup will show either just the GIHWR or both the GIHWR and the grade or rating. If there are no GIHWR values, like with Charitable Levy, the popup won’t display this additional table. The items in the list are sorted by sample size in descending order, which is what I believe LG is doing.

@unrealities, @JaggerMin What are your thoughts on this?

![image](https://github.com/user-attachments/assets/314dc027-8cad-4d07-b4e5-11a9c1c591bc)

![image](https://github.com/user-attachments/assets/e089b472-9080-4857-b3df-5b3c16102b6e)

![image](https://github.com/user-attachments/assets/9c7c98a0-2152-414f-8163-b355e9162849)

![image](https://github.com/user-attachments/assets/9c342f72-e044-4eaa-a301-e67634eecadf)

**Reply by unrealities** [reply]
Looks cool to me!

**Reply by JaggerMin** [reply]
It seems like a neat feature!

**Comment by unrealities** [comment]
Released in v3.24
